README.md

```diff
@@ -1,11 +1,83 @@
-<h1 align="center">kmsconfig</h1>
+<h1 align="center">go-kmsconfig</h1>
 
 <p align="center">
   JSON config with KMS encryption support.
 </p>
 
+## Setup
+
+`go-kmsconfig` expects the following config structure:
+
+```
+- config
+-- staging.json
+-- qa.json
+-- live.json
+```
+
+An exmaple of a config file looks like:
+
+```json
+
+{
+  "app": { // Top level node name
```

Comment:
Comments don't look good in Github

![image](https://user-images.githubusercontent.com/2796074/30479072-7bf90d1c-9a0b-11e7-940d-f6024400674c.png)

Remove...